pkg/sfu/downtrack.go

```diff
@@ -133,16 +138,13 @@ type DownTrack struct {
 	lossFraction atomicUint8
 
 	// Debug info
-	lastPli                         atomicInt64
-	lastRTP                         atomicInt64
-	pktsMuted                       atomicUint32
-	pktsDropped                     atomicUint32
-	pktsBandwidthConstrainedDropped atomicUint32
+	lastPli     atomicInt64
+	lastRTP     atomicInt64
+	pktsMuted   atomicUint32
+	pktsDropped atomicUint32
 
 	maxPacketTs uint32
 
-	bandwidthConstrainedMuted atomicBool
```

Comment:
Removing this and managing everything through spatial/temporal layer values. Introduced `InvalidSpatial/TemporalLayer` to signify something is bandwidth constraint muted.